.github/ISSUES_BACKLOG.md

```
@@ -0,0 +1,233 @@
+# Issues Backlog
+
+> このファイルは、GitHub Issues への登録前のバックログです。`bash scripts/create_issues.sh --create` で一括登録できます（事前に `gh auth login` が必要）。
+
+---
+
+## Issue: Home画面 左サイドショートカット列の実装
+
+- Title: feat(UI): Home画面に左サイドショートカット列を実装
+- Body:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)